unit_tests/test_users.py

```diff
@@ -33,6 +33,10 @@ def test_configure_admin_custom_password(self):
         If a password is provided, it's used to configure the admin user.
         """
         self.fakes.juju.config["password"] = "x"
+        # TODO figure out why self.fakes.juju setting is not reliably working.
```

Comment:
The problem here is I expected from looking at other code that self.fakes.juju.config["password"] would set the config option for the password but in my testing it did nothing it wasn't until I used hookenv.config() to get the config and set it that way that the test begun to work.